### PR TITLE
More fixes for PEP8 compliance

### PR DESCRIPTION
**Pull request by zuphilip**
These fixes are IMO more advanced. Please look carefully and we also should probably make some manual tests.

### CODE REVIEW

File: hocr-check [hunk 1]
```diff
@@ -83,7 +83,8 @@ def mostly_nonoverlapping(boxes, significant_overlap=0.2):
 ################################################################
 
 parser = argparse.ArgumentParser(
-    description="Check the given file for conformance with the hOCR format spec"
+    description=("Check the given file for conformance with the hOCR"
+                 "format spec")
```
Comment (stweil) [inline hocr-check]: Here the blank got lost ("hOCRformat").

File: hocr-cut [hunk 2]
```diff
@@ -28,8 +28,9 @@ def get_bbox(node):
 
 
 parser = argparse.ArgumentParser(
-    description=
-    'Cut a page (horizontally) into two pages in the middle such that the most of the bounding boxes are separated nicely, e.g. cutting double pages or double columns'
+    description=('Cut a page (horizontally) into two pages in the middle'
+                 'such that the most of the bounding boxes are separated'
+                 'nicely, e.g. cutting double pages or double columns')
```
Comment (stweil) [inline hocr-cut]: Same problem as above, more follow.

File: hocr-check (hunk 1 above)
Comment (zuphilip) [reply hocr-check]: Okay, do you want to fix these missing spaces?

**Comment by zuphilip** [comment]
@stweil The spacing issues should be fixed now. Please have another look.

**Comment by stweil** [comment]
Thank you. Travis now also checks the style.

**Pull request merged**